.gitignore

```diff
@@ -62,5 +62,4 @@ jacoco.exec
 results/
 test_artifacts/
 google-cloud-sdk/
-*.jar
```

Comment:
@pocmo we talked about this briefly, but I think I misunderstood. Wouldn't we want to force add only the ADM jar and keep this gitignore filter here so we don't mistakenly commit other jars that shouldn't be here?

Reply:
Never force add anything. Fix the gitignore filter instead. This is way to broad. Whether we want to add a jar or not always depends on the case.
This was added for ignoring some UI test jars, and the fix in Fenix is here - let's do the same:
https://github.com/mozilla-mobile/fenix/pull/4088/commits/fe2d589171a67d44150c89a288f6331b166a95a8